index.html

```diff
@@ -25,8 +25,8 @@
               </li>
               <li class="navigation-list-item">
                 <a class="navigation-list__link" href="catalog.html"
-                  >Гостиницы</a
-                >
+                  >Гостиницы
+                </a>
               </li>
             </ul>
 
```

Comment:
пропустил этот момент
когда есть явный заголовок для секции не надо придумывать свой
Заинтересовались должно быть h2
```suggestion
<section class="hotel-booking">
          <h2 class="visually-hidden">Поиск гостиницы.</h2>
          <div class="hotel-wrapper">
            <p class="hotel-booking__subtitle"><b>Заинтересовались?</b></p>
            <p class="hotel-booking__text">
              Укажите предполагаемые даты поездки, и мы покажем вам лучшие
              предложения гостиниц в Седоне
            </p>
            <button class="hotel-booking__search-hotels-button" type="button">
              Поиск гостиницы в Седоне
            </button>
          </div>
        </section>
```